kfdefs/overlays/moc/zero/opf-monitoring/scrape-configs/additional-scrape-configs.yaml

```diff
@@ -19,3 +19,50 @@ stringData:
       - targets: ["argocd-repo-server-metrics-argocd.apps.moc-infra.massopen.cloud"]
         labels:
```

Comment:
@harshad16 you can give each job a unique label so it can be queried easily.

Reply:
yes, just added that labels for each job, PTAL